cmd/shortener/server.go

```diff
@@ -1,17 +1,88 @@
 package main
 
 import (
+	"flag"
 	log "github.com/sirupsen/logrus"
-	"go-url-shortener/configs"
 	"go-url-shortener/internal/handlers"
+	"go-url-shortener/internal/storage"
 	"net/http"
+	"os"
 )
 
+const (
+	addrKey         = "SERVER_ADDRESS"
+	baseKey         = "BASE_URL"
+	storageFileName = "FILE_STORAGE_PATH"
+)
+
+var config struct {
+	addr     string
```

Comment:
можно проставить теги всем полям, тогда можно будет использовать env.Parse
```suggestion
	addr     string     `env:"SERVER_ADDRESS"`
```